netutils/iperf/Make.defs

```diff
@@ -0,0 +1,20 @@
+############################################################################
+# Copyright 2020 Espressif Systems (Shanghai) PTE LTD
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+############################################################################
+
+ifneq ($(CONFIG_EXAMPLES_IPERF),)
+CONFIGURED_APPS += $(APPDIR)/external/iperf
```

Comment:
Similar as above, paths need to be adapted.

Reply:
It has been modified.